recipes/minizip-ng/all/conandata.yml

```diff
@@ -28,10 +28,6 @@ patches:
     - patch_file: "patches/3.0.8-0001-fix-cmake-project.patch"
       patch_description: "CMake: declare project() sooner"
       patch_type: "conan"
-    - patch_file: "patches/3.0.6-0002-fix-lzma-libdir.patch"
```

Comment:
Was this a copy paste error?

Reply:
probably. PR in "patch_source" was merged in 3.0.8 release